static/js/settings_org.js

```diff
@@ -138,55 +172,26 @@ exports.set_up = function () {
         // TODO: We actually have three forms named admin-realm-form.  We really
         //       should break out three separate forms.
 
+        for (var k1 in property_types) {
+            if (property_types.hasOwnProperty(k1)) {
+                $("#admin-realm-"+k1.split('_').join('-')+"-status").expectOne().hide();
```

Comment:
Also, since this is a repeated and messy pattern, I'd give it a name, e.g. "property_type_status_element(k1).hide()"